en/organisation/sammansattning/body.md

```diff
@@ -1,179 +1,100 @@
-Members of the Board
---------------
+# Members of the Board
 
 The D-directorate is the board of the Computer Science chapter and consists of seven members.
 They are the Chairman, Vice Chairman, Treasurer, secretary, and three more board members that are responsible
 for educational issues, student welfare, and business relations. Together they arrange Chapter Meetings, make sure
 the decisions at chapter meetings are executed, coordinate different parts of the chapter, and make strategic and long term plans.
 
-Each board member is the contact person in the board for some of out elected volunteers.
+Each board member is the contact person in the board for some of out elected volunteers a list can be found [here](https://dsekt.se/kontaktfunktionarare).
 
-## The Chairman - David Puustinen
+## The Chairman - Pontus Söderlund
 
 The Chairman is in charge of the D-directorate and the contact with THS and other chapters.
 The Chairman and the Vice Chairman work in close cooperation to create a plan and a vision with the rest of the board.
 
 In addition to this The Chairman is an authorized signatory for the chapter and handles alcohol related questions.
 
-__Contact__
+**Contact**
```

Comment:
Is this your autoformatter going crazy?
I don't care really, just wondering about the diff.